build/system.gyp

```diff
@@ -11,6 +11,19 @@
   },
   'targets' : [
     {
+      'target_name': 'linux_libnotify',
```

Comment:
Because, maybe, others might be confused and try to depend on it if it's named as 'libnotify'. This target is only to import 'libnotify' for 'linux'.